agentops/__init__.py

```diff
@@ -12,13 +13,21 @@
 
 
 def init(api_key: Optional[str] = None,
+         parent_key: Optional[str] = None,
          tags: Optional[List[str]] = None,
          endpoint: Optional[str] = environ.get('AGENTOPS_API_ENDPOINT', 'https://api.agentops.ai'),
          max_wait_time: Optional[int] = 1000,
          max_queue_size: Optional[int] = 100,
          override=True,
          auto_start_session=True):
-    Client(api_key, tags, endpoint, max_wait_time, max_queue_size, override, auto_start_session)
+
+    if not parent_key:
+        parent_key = environ.get('AGENTOPS_PARENT_KEY', None)
+
+    if not endpoint:
+        endpoint = environ.get('AGENTOPS_API_ENDPOINT', 'https://api.agentops.ai')
```

Comment:
Not needed

Reply:
it kind of is because if .init() has no params, the optional logic makes the value `None`. But then the Client() constructor is receiving None for the params which does not trigger the optional default logic. So we need to handle that case separately